generated/nirfsg/nirfsg/session.py

```diff
@@ -2040,66 +2022,66 @@ class _SessionBase(object):
     | ScriptTrigDigLevelActiveLevel.LOW  | 9001 (0x2329) | Trigger when the digital trigger signal is low.  |
     +------------------------------------+---------------+--------------------------------------------------+
     '''
-    exported_pulse_modulation_event_output_terminal = _attributes.AttributeEnum(_attributes.AttributeViString, enums.PulseModulationOutputTerm, 1150309)
-    '''Type: enums.PulseModulationOutputTerm
+    exported_pulse_modulation_event_output_terminal = _attributes.AttributeViString(1150309)
+    '''Type: str
 
     Specifies the destination terminal for exporting the Pulse Modulation Event. The Pulse Modulation Event tracks the RF Envelope when Pulse Modulation is Enabled. If this property is set to a value other than `do not export str`, calling NI-RFSG Commit will cause the output terminal to be pulled to the logic level that is the inverse of `exported pulse modulation event active level`. You can tri-state this terminal by setting this property to `do not export str` or by calling `niRFSG Reset`. To set this property, the NI-RFSG device must be in the Configuration state.
 
-                    **Default Value:** PulseModulationOutputTerm.PULSE_OUT
+                    **Default Value:** NIRFSG_VAL_PULSE_OUT_STR
```

Comment:
Another one that you deleted. Time to audit for the use of these deleted enums in descriptions.